random_generation.c

```diff
@@ -3,31 +3,33 @@
 #include <time.h>
 
 
-int cong_generator()
-{
-    static int nmbr;
-    int a,c,m;
-    a = 7;
-    c = 2;
-    m = 1000;
-    nmbr = (a*nmbr + c) % m;
-    return nmbr;
-}
+// int cong_generator()
```

Comment:
Minor: please remove this code if it's unused. You can dig into the commit history in case you'll need it in the future.